src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatClientExtensions.cs

```diff
@@ -120,6 +121,34 @@ public static Task<ChatResponse> GetResponseAsync(
         return client.GetResponseAsync([chatMessage], options, cancellationToken);
     }
 
+    /// <summary>Gets a background response identified by the specified continuation token.</summary>
+    /// <param name="client">The chat client.</param>
+    /// <param name="continuationToken">The continuation token.</param>
+    /// <param name="options">The chat options to configure the request.</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <returns>The background response.</returns>
```

Comment:
Can you elaborate on what "background response" means?  And is it necessarily a "background" response? Does the abstraction contract say that a continuation token is only ever provided if AllowBackground is true?